packages/react-components/react-menu/src/components/MenuItemSwitch/MenuItemSwitch.types.ts

```diff
@@ -0,0 +1,21 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { MenuItemSlots } from '../MenuItem/MenuItem.types';
+import { MenuItemSelectableState } from '../../selectable/types';
+import { MenuItemCheckboxProps } from '../MenuItemCheckbox/MenuItemCheckbox.types';
+
+export type MenuItemSwitchSlots = Pick<MenuItemSlots, 'root' | 'content' | 'secondaryContent' | 'icon'> & {
+  switchIndicator?: Slot<'span'>;
```

Comment:
```suggestion
  switch?: Slot<'span'>;
```

Just `switch`? `MenuItemCheckbox` has just `checkmark`